site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>I don't know if this is possible given we are limiting outputs - but if it is, can you demo each of these functions as you build them?</p><p> </p><p>Showing the call &amp; output is a helpful pedagogical tool.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>We are publishing without the outputs. The previous version of this notebook had example outputs, but after consulting with the OSS team, we have removed the outputs.</p>